### ISSUE TITLE
HDMI CEC Documentation out of date

### ISSUE THREAD

**Issue by mhirsch** [home-assistant/home-assistant.github.io#8014]
The [HDMI CEC documentation](https://www.home-assistant.io/components/hdmi_cec/) seems out of date. There's a link to the default virtual environment of a [HASSbian Image for Raspberry Pi](https://www.home-assistant.io/getting-started/installation-raspberry-pi-image/) which is a 404. It also says to symlink to the installed version of _cec.so, but in raspbian the python-libcec package doesn't have this file. If it needs to be built separately it would be helpful to mention it.

Thanks!

**Comment by nwesterhausen** [comment]
I looked at the page HDME CEC doc page. Following the link for the virtualenv installation instructions sent me to a redirect to the actual page. I will fix the link.

From what I read up on in the doc, it seems if your Home  Assistant venv can utilize the python-libcec package you are OK, not sure based on your post if you have tried this and it doesn't work or if you were reading up on this before trying it. I would suggest trying it, trying it with the venv created with the `--system-site-packages` flag (as suggested in the doc), and if things still aren't working to reach out on the forums or chat for more help. If the files aren't there for symlinking we should figure out what the proper way to get things working is and then update the docs after figuring that out.

**Comment by mhirsch** [comment]
Thank you! I have already tried it, but my problem is that I'm using raspbian and the `libcec-dev` they ship seems to not include the python bindings, among other problems. I filed a bug with raspbian (https://bugs.launchpad.net/raspbian/+bug/1804365) about that and other issues related to the `libcec-dev` package (https://bugs.launchpad.net/raspbian/+bug/1802820). Is there a version of `libcec` or `libcec-dev` that contains the python bindings that's used with HASSbian that's available somewhere?

Thanks for fixing the link.
